mypy/nodes.py

```diff
@@ -2500,6 +2500,7 @@ class SymbolTableNode:
                  'implicit',
                  'is_aliasing',
                  'alias_name',
+                 'no_serialize'
```

Comment:
Trailing comma. Also add a comment to the end of the docstring about it please.

Reply:
Sorry, already merged before seeing your comment. Will fix this later.